drivers/Makefile.include

```diff
@@ -18,6 +18,14 @@ ifneq (,$(filter apa102,$(USEMODULE)))
   USEMODULE_INCLUDES += $(RIOTBASE)/drivers/apa102/include
 endif
 
+ifneq (,$(filter at24cxxx,$(USEMODULE)))
+  USEMODULE_INCLUDES += $(RIOTBASE)/drivers/at24cxxx/include
+endif
+
+ifneq (,$(filter at24cxxx_eeprom_wrapper,$(USEMODULE)))
+  CFLAGS += -DEEPROM_SIZE=0x8000
```

Comment:
I would move this block in the `at24cxxx` one above.

Also could this be mapped with `AT24CXXX_EEPROM_SIZE` params ?

Reply:
This would be super nice. But not without adding `-include at24cxxxx_wrapper.h` to the cflags, as users of the wrapper are unaware of the `at24cxxx` driver will not include the right header files themselves.

Reply:
> How about
> 
> ```makefile
>   ifneq (,$(filter at24c256%,$(USEMODULE)))
>     # Size is 256 Kib (= 32 KiB) on AT24C256/AT24C256C
>     CFLAGS += -DEEPROM_SIZE=0x8000
>   else
>     # Size is 128 KiB (= 16 KiB) at AT24C128
>     CFLAGS += -DEEPROM_SIZE=0x4000
>   endif
> ```

> I would move this block in the at24cxxx one above.

Like that?
```
ifneq (,$(filter at24cxxx,$(USEMODULE)))
  USEMODULE_INCLUDES += $(RIOTBASE)/drivers/at24cxxx/include
  ifneq (,$(filter at24cxxx_eeprom_wrapper,$(USEMODULE)))
    ifneq (,$(filter at24c256%,$(USEMODULE)))
      # Size is 256 Kib (= 32 KiB) on AT24C256/AT24C256C
      CFLAGS += -DEEPROM_SIZE=0x8000
    else
      # Size is 128 KiB (= 16 KiB) at AT24C128
      CFLAGS += -DEEPROM_SIZE=0x4000
    endif
  endif
endif
```